vignettes/complete-multiverse-analysis.Rmd

```diff
@@ -270,8 +266,6 @@ M %>% execute_default( universe_parameter_assignment ) %>%
 ```{r, eval = FALSE}
 M %>%
   execute_multiverse()
-
 M@multiverse_table$results[[2]]$df
```

Comment:
this doesn't look like user-facing code to me. should use `multiverse_table(M)` ?